auslib/admin/views/forms.py

```diff
@@ -374,3 +374,101 @@ class EditScheduledChangeExistingReleaseForm(ScheduledChangeTimeForm):
     data = JSONStringField(None, 'Data', validators=[Optional()], widget=FileInput())
     data_version = IntegerField('data_version', widget=HiddenInput())
     sc_data_version = IntegerField('sc_data_version', validators=[InputRequired()], widget=HiddenInput())
+
+
+class ProductRequiredSignoffForm(Form):
+    product = StringField('Product', validators=[Length(0, 15), InputRequired()])
```

Comment:
Yeah, I hear you. I'm sure this is easy if we were to switch to Django or some other full stack ORM...but that's not going to happen. We might be able to reference database tables here instead of copying the value over - I'm not sure how easy it is to get a reference to things like column length. Sounds like something that would be good to address as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1336452, unless you think it's pressing.